atom/browser/native_window_mac.mm

```diff
@@ -933,13 +934,17 @@ void ViewDidMoveToSuperview(NSView* self, SEL _cmd) {
     if (!fullscreen_window_title()) {
       // Restore the titlebar
       SetStyleMask(true, NSTitledWindowMask);
-    } else {
-      // Show the window buttons
-      [[window standardWindowButton:NSWindowZoomButton] setHidden:NO];
-      [[window standardWindowButton:NSWindowMiniaturizeButton] setHidden:NO];
-      [[window standardWindowButton:NSWindowCloseButton] setHidden:NO];
     }
 
+    // Restore window controls visibility state
+    const bool window_button_visible = window_button_visibility_.value_or(true);
```

Comment:
The double negative in this code block confused me a bit, can we make this `window_buttons_hidden` with a default of `false`

Reply:
Done!